changelog.md

```diff
@@ -1,3 +1,32 @@
+## 3.5.0 / 2021-xx-xx
+
+### New Features
+
+- All users can now set a public user profil
+
+### Fixed Issues
+
+- Frontend: [#3578](https://github.com/tracim/tracim/issues/3578),
+[#3785](https://github.com/tracim/tracim/issues/3785),
+[#3790](https://github.com/tracim/tracim/issues/3790),
+[#3989](https://github.com/tracim/tracim/issues/3989),
+[#3999](https://github.com/tracim/tracim/issues/3999)
+- Other: [#4062](https://github.com/tracim/tracim/issues/4062)
+
+### Known Issues
+
+- SQLite: IDs of objects can be re-used on database (issue [#4016](https://github.com/tracim/tracim/issues/4016))
```

Comment:
the issue is not that id can be reused (this is the reason for the bug). what is the bug symptom?

Reply:
I not know how to explain that in 2 lines.

Reply:
I purpose something like this 
~~~
SQLite: if you delete one user in the database and create a new one, SQlite can assign the same user_id like visible for the old one.
In this case, some information in Tracim is not referer to the good user.
~~~

Thats ok for you @lebouquetin @grignards

Reply:
SQLite: After deleting a content/user/space directly in the database its id can be re-used (see details [here](https://sqlite.org/autoinc.html)). This can lead Tracim to display information referring to the wrong object in the notifications and activity feed.

Reply:
Done